modules/combined/tests/check_error/contact_displacements.i

```diff
@@ -1,37 +1,23 @@
-[Mesh]#Comment
+[Mesh]
   file = cube.e
-#  displacements = 'disp_x disp_y disp_z'
-[] # Mesh
+[]
 
 [Variables]
-
   [./disp_x]
-    order = FIRST
-    family = LAGRANGE
   [../]
-
   [./disp_y]
-    order = FIRST
-    family = LAGRANGE
   [../]
-
   [./disp_z]
-    order = FIRST
-    family = LAGRANGE
   [../]
+[]
 
-[] # Variables
-
-[SolidMechanics]
-  [./solid]
-    disp_x = disp_x
-    disp_y = disp_y
-    disp_z = disp_z
+[Kernels]
+  [./TensorMechanics]
+    displacements = 'disp_x disp_y disp_z'
```

Comment:
I looked at the ContactAction.C, and turned out that the intend of this test is to check whether the displacements is set in the mesh block.